### DISCUSSION TITLE
Is there a way for renovate to update version custom github actions that's internal to an organisation ?

### DISCUSSION THREAD

**Discussion by akshay5995** [renovatebot/renovate#18530] [Request Help] [answered]
I'm trying to find docs about having renovate update [custom github action](https://dev.to/n3wt0n/finally-custom-github-actions-in-internal-repos-4l91) internal to the org.

I was not able to find any doc related to this. Please help 🥺

**Comment by viceice** [comment, marked as answer]
sure, as long as the bot has access to the private repo.

**Reply by akshay5995** [reply]
Thank you!
